src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import MainContent from './main-content/MainContent';
+import SecondaryContent from './secondary-content/SecondaryContent';
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as chevronLeftIcon } from 'static/img/shared/chevron-left.svg';
+import { ReactComponent as chevronRightIcon } from 'static/img/shared/chevron-right.svg';
+
+import { Status } from 'src/shared/types/status';
+
+import styles from './TranscriptsTable.scss';
+
+const TranscriptsTable = () => {
+  const [paginationNumber, setPaginationNumber] = useState(1);
+
+  const gotoPrevious = () => {
+    if (paginationNumber > 1) {
+      setPaginationNumber(paginationNumber - 1);
+    }
+  };
+
+  const gotoNext = () => {
+    if (paginationNumber < 3) {
+      setPaginationNumber(paginationNumber + 1);
+    }
+  };
```

Comment:
`gotoPrevious`, `gotoNext` → `previousPage`, `nextPage`

Reply:
Don't those look more like variables than listeners?

Reply:
They do, yes. What I didn't like about the previous names was that they made me ask the question "previous what". `toPreviousPanel`, `toNextPanel` perhaps?

Reply:
Yes, they look much better.

Reply:
:heavy_check_mark: